telegram/bot.py

```diff
@@ -4971,6 +5162,12 @@ def __hash__(self) -> int:
     """Alias for :attr:`set_chat_administrator_custom_title`"""
     exportChatInviteLink = export_chat_invite_link
     """Alias for :attr:`export_chat_invite_link`"""
+    createChatInviteLink = create_chat_invite_link
```

Comment:
Do we want versioning directives here as well?

Reply:
I'd say no, since original method already has it

Reply:
+1, that should be enough and would blow up this alias section needlessly imo